src/Microsoft.Identity.Web/AzureADB2COpenIDConnectEventHandlers.cs

```diff
@@ -31,7 +31,7 @@ public AzureADB2COpenIDConnectEventHandlers(
 
         public MicrosoftIdentityOptions Options { get; }
 
-        public Task OnRedirectToIdentityProvider(RedirectContext context)
```

Comment:
Did you try pack the project?
Didn't the public analyzers didn't complain.

These events are defined by ASP.NET Core? do they have the Async in their name?

Reply:
I did pack it and there were no complaints. I must have missed this because it's an internal class. I'll keep the names and add a suppression.